igvm_defs/src/lib.rs

```diff
@@ -1037,7 +1038,7 @@ pub struct IGVM_VHS_MEMORY_MAP_ENTRY {
 
 /// The signature algorithm to use for VBS digest.
 #[open_enum]
-#[derive(AsBytes, FromBytes, Debug, Clone, Copy, PartialEq, Eq)]
+#[derive(AsBytes, FromBytes, FromZeroes, Debug, Clone, Copy, PartialEq, Eq)]
 #[repr(u32)]
 pub enum VbsDigestAlgorithm {
```

Comment:
I don't understand how "FromZeroes" makes sense here - the enum only has one value, and it isn't zero?

Reply:
FromBytes requires FromZeroes automatically in the drive macro in the latest version. Otherwise, it gives compilation error.

Reply:
This looks like a similar story to what happened with the mshv crate in the past: https://github.com/rust-vmm/mshv/commit/98ecc1dca25bcd333a4933a71b8ad240f62b9f86

Reply:
Interesting requirement. We probably should define a value of zero here then as INVALID.